src/include/sof/trace/trace.h

```diff
@@ -170,11 +170,22 @@ void trace_log(bool send_atomic, const void *log_entry,
 unsupported_amount_of_params_in_trace_event\
 _thrown_from_macro_BASE_LOG_in_trace_h
 
+
+#define CT_ASSERT(COND, MESSAGE) \
+	((void)sizeof(char[1 - 2*!(COND)]))
```

Comment:
if you do another iteration of the series, maybe you could remove the superfluous empty line above and add spaces around `*`

Reply:
done